pkg/drivers/vfkit/iso.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
2024 here and other files

Reply:
We don't normally update old copyright years, but keep when it was written?

Reply:
i.e. the file was copied from pkg/hyperkit

Reply:
Save this for the refactoring